proxy/envoy/policy.go

```diff
@@ -21,8 +21,9 @@ import (
 	"sort"
 	"strings"
 
+	proxyconfig "istio.io/api/proxy/v1/config"
+
```

Comment:
Why the extra newline between proxyconfig and the other `istio.io` imports? They are both from the same base import path which is traditionally grouped together.